psycopg_pool/psycopg_pool/null_pool_async.py

```diff
@@ -7,12 +7,18 @@
 from __future__ import annotations
 
 import logging
-from typing import Any, cast
+from typing import cast
 
 from psycopg import AsyncConnection
 from psycopg.pq import TransactionStatus
 
-from .abc import ACT, AsyncConnectFailedCB, AsyncConnectionCB
+from .abc import (
+    ACT,
+    AsyncConnectFailedCB,
+    AsyncConnectionCB,
+    AsyncConninfoParam,
+    AsyncKwargsParam,
+)
```

Comment:
nit: Uhm... hasn't isort fixed/complained about this import style? It should be more like:
```suggestion
from .abc import ACT, AsyncConnectFailedCB, AsyncConnectionCB
from .abc import AsyncConninfoParam, AsyncKwargsParam
```